### DISCUSSION TITLE
How to request qbittorrent's api across domains?

### DISCUSSION THREAD

**Discussion by wintsa123** [VueTorrent/VueTorrent#1599] [Q&A] [answered]
I'm working on a navigation homepage and would like to get info on qbittorrnt, but after using some tool libraries, I found that the requests are cross-domain, I tried to learn VueTorrent at vite to turn on the cross-domain settings, but it didn't work!
![image](https://github.com/VueTorrent/VueTorrent/assets/47288630/f6dc1c38-ab52-4563-a56d-e99f047c78d3)

**Comment by Larsluph** [comment, marked as answer]
Everything's mentioned in the readme. To make the proxy work, you MUST disable security features to use the API outside of qbittorrent's domain.

"CSRF protection" and "Host header verification" should be enough to make it work.
